layouts/index.html

```diff
@@ -235,7 +241,7 @@ <h3>Interested in sponsorship?</h3>
                     <img src="assets/images/contribute.png" width=300px>
                     <h3>Want to contribute?</h3>
                     <p>We're working on exciting things for the Rovercode platform, and it's all open-source.</p>
-                    <a href="https://contributor-docs.rovercode.com" class="ui button">Check out the docs</a>
+                    <a href="https://github.com/rovercode" class="ui button">Check us out on GitHub</a>
```

Comment:
I know just dumping someone on our github repos list is not super helpful, but I think it's better than out out-of-date contributor docs for now.